client/src/pages/AdminEditTestimonialsPage.tsx

```diff
@@ -28,7 +31,8 @@ export const EditTestimonialsContext = React.createContext<{
         careClosetVisitsStatError: "",
         diapersDistributedStatError: "",
         donorHomepageConfig: DonorHomepageConfig,
-        regularDonorsStatError: ""
+        regularDonorsStatError: "",
+        editingClientStory: false
     },
```

Comment:
you can change this to `formState: initialFormState` since the objects are the same